# Parameters for pure pursuit controller
L: 0.47    # Length of car (m)
lfw: 0.1675 # distance between the front of the vehicle and its center of gravity
goal_radius: 0.5 # unit: m
controller_freq: 40 # unit: Hz
base_angle: 0.0 # neutral point of servo (rad)
cmd_vel_mode: true # whether or not publishing cmd_vel
smooth_accel: true # whether or not smoothing the acceleration of car
speed_incremental: 0.1 # speed incremental value (discrete acceleraton), unit: m/s
debug_mode: false
predict_time: 0.0 # predict vehicle movement according to kinematic model, to avoid control delay
min_velocity: 2.0
